pkg/controllers/disruption/emptiness.go

```diff
@@ -50,7 +50,6 @@ func (e *Emptiness) ComputeCommand(_ context.Context, candidates ...*Candidate)
 	emptyCandidates := lo.Filter(candidates, func(cn *Candidate, _ int) bool {
 		return cn.NodeClaim.DeletionTimestamp.IsZero() && len(cn.pods) == 0
 	})
-	deprovisioningEligibleMachinesGauge.WithLabelValues(e.Type()).Set(float64(len(candidates)))
```

Comment:
I'm a bit cautious to see all these metrics removed w/o fanfare.

Reply:
Same answer as above.